Inter-Research Science Center.js

```diff
@@ -91,8 +91,8 @@ function scrape(doc, url){
 		item.journalAbbreviation = journals[jour][1];
 		item.abstractNote = Zotero.Utilities.trimInternal(doc.evaluate('//p[@class="abstract_block"]', doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent);
 		var authors = Zotero.Utilities.trimInternal(doc.evaluate('//div[@class="bb"]/h3', doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent).split(/,\s+/);
-		for each (var aut in authors) {
-			aut = aut.replace(/[^\w^\s^\.]/g, "").replace(/\d/g, "");
+		for (var i=0; i<authors.length; i++) {
+			var aut = authors[i].replace(/[^\w^\s^\.]/g, "").replace(/\d/g, "");
```

Comment:
I don't understand. Replaced with empty string? Only this one replace or also the second replace from the line above? Should I really change something here? I just undo all space improvements ...